kinematics/tests/test_kinematics_initialization.py

```diff
@@ -1,9 +1,9 @@
 from pytest import raises
-from MotionPlanning.kinematics.kinematics_solvers import KinematicsSolver
-from MotionPlanning.kinematics.kinematics_exceptions import InvalidInitVector
 import numpy as np
 from numpy.testing import assert_allclose
-from MotionPlanning.kinematics.tests import ATOL, RTOL
+from kinematics.kinematics_solvers import KinematicsSolver
+from kinematics.kinematics_exceptions import InvalidInitVector
```

Comment:
To coś rzuca błędami przy pyteście
![image](https://github.com/HexapodBionik/MotionPlanning/assets/128425202/57bef125-a150-491e-ae0b-43a51ffa4e1e)